src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,48 @@
 import React from 'react';
+import { CommentList } from '../CommentList';
+import { Post } from '../../types/post';
+import { UserInfo } from '../UserInfo';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
-);
+import './PostInfo.scss';
+
+type Props = {
+  post: Post,
+};
+
+export const PostInfo: React.FC<Props> = ({ post }) => {
+  const {
+    user,
+    comments,
+    title,
+    body,
+  } = post;
+
+  return (
+    <>
+      <div className="PostInfo">
+        <div className="PostInfo__header">
+          <h3 className="PostInfo__title">{title}</h3>
+
+          <p>
+            {' Posted by  '}
```

Comment:
Remove empty lines and format them (like in the previous cases, when I wrote the comments)